funcframework/framework.go

```diff
@@ -97,36 +104,44 @@ func registerEventFunction(path string, fn interface{}, h *http.ServeMux) {
 		}()
 		handleEventFunction(w, r, fn)
 	})
+	return nil
+}
+
+func registerCloudEventFunction(path string, fn func(context.Context, cloudevents.Event), h *http.ServeMux) error {
```

Comment:
looks like this is coming from the sample: https://github.com/cloudevents/sdk-go/blob/master/samples/http/receiver-direct/main.go